src/createApp.ts

```diff
@@ -569,17 +569,28 @@ function addIdentifier(app: App, id: Identifier) {
 
 function createCustomWidget(app: App, id: string) {
 	const { registryProvider, defaultWidgetStore } = app;
+	let factoryHandle: Handle;
 	// istanbul ignore if
 	if (!defaultWidgetStore) {
 		throw new Error('A default widget store must be configured in order to create custom widgets');
 	}
 
 	return defaultWidgetStore.get(id).then((state: any) => {
-		const options: WidgetFactoryOptions = { id, stateFrom: defaultWidgetStore, registryProvider, state };
 		const customFactory = customElementFactories.get(app).get(state.type);
```

Comment:
Just realizing now it'd be nice if this checks whether `state.type` even has a (string) value.